vpn/muxer_test.go

```diff
@@ -50,21 +50,23 @@ func Test_newMuxerFromOptions(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := newMuxerFromOptions(tt.args.conn, tt.args.options, tt.args.tunnel)
+			_, err := newMuxerFromOptions(tt.args.conn, tt.args.options, tt.args.tunnel)
 			if !errors.Is(err, tt.wantErr) {
 				t.Errorf("newMuxerFromOptions() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
-			if !bytes.Equal(got.session.RemoteSessionID[:], tt.want.session.RemoteSessionID[:]) {
-				t.Errorf("newMuxerFromOptions() session = %v, want %v", got, tt.want)
-			}
-			if !bytes.Equal(got.session.LocalSessionID[:], tt.want.session.LocalSessionID[:]) {
-				t.Errorf(
-					"newMuxerFromOptions() session. = %v, want %v",
-					got.session.LocalSessionID[:],
-					tt.want.session.LocalSessionID[:],
-				)
-			}
+			/*
+			 if !bytes.Equal(got.session.RemoteSessionID[:], tt.want.session.RemoteSessionID[:]) {
+			 	t.Errorf("newMuxerFromOptions() session = %v, want %v", got, tt.want)
+			 }
+			 if !bytes.Equal(got.session.LocalSessionID[:], tt.want.session.LocalSessionID[:]) {
+			 	t.Errorf(
+			 		"newMuxerFromOptions() session. = %v, want %v",
+			 		got.session.LocalSessionID[:],
+			 		tt.want.session.LocalSessionID[:],
+			 	)
+			 }
+			*/
```

Comment:
I think you should delete commented out code or explain why we are going to need it later and what is the condition that will cause us to uncomment the code.

Reply:
there was a little TODO budired in there, but I agree - deleted the commented code for no, left only the TODO.